ui/src/app/job-list/job-list.module.ts

```diff
@@ -15,7 +15,7 @@ import {
   MatSelectModule,
   MatSortModule,
   MatTableModule,
-  MatTooltipModule
+  MatTooltipModule,
```

Comment:
I would probably just revert this file since it's not relevant to the change anymore